lib/publiccloud/provider.pm

```diff
@@ -625,7 +625,8 @@ sub terraform_param_tags
     my $tags = {
         openqa_ttl => get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300),
         openqa_var_JOB_ID => get_current_job_id(),
-        openqa_var_NAME => get_var(NAME => '')
+        openqa_var_NAME => get_var(NAME => ''),
+        openqa_var_SERVER => get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'))
```

Comment:
`# note: OPENQA_URL is sometimes just the hostname (eg. [e212.suse.de](http://e212.suse.de/)) but might be a proper URL`

so what you have wrote above is not enough . we need somewhere to have logic which deciding if it will be `http://openqa.server` or it will be `https://openqa.server`

Reply:
The logic for determining the scheme should be in pcw.